hbase-server/src/main/resources/hbase-webapps/master/hbck.jsp

```diff
@@ -0,0 +1,139 @@
+<%--
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+--%>
+<%@ page contentType="text/html;charset=UTF-8"
+         import="java.util.List"
+         import="java.util.Map"
+         import="java.util.stream.Collectors"
+%>
+<%@ page import="org.apache.hadoop.hbase.master.HbckChecker" %>
+<%@ page import="org.apache.hadoop.hbase.master.HMaster" %>
+<%@ page import="org.apache.hadoop.hbase.ServerName" %>
+<%@ page import="org.apache.hadoop.hbase.util.Pair" %>
+<%
+  HMaster master = (HMaster) getServletContext().getAttribute(HMaster.MASTER);
+  pageContext.setAttribute("pageTitle", "HBase Master HBCK Report: " + master.getServerName());
+  HbckChecker hbckChecker = master.getHbckChecker();
+  Map<String, Pair<ServerName, List<ServerName>>> inconsistentRegions = null;
+  Map<String, ServerName> orphanRegionsOnRS = null;
+  List<String> orphanRegionsOnFS = null;
+  if (hbckChecker != null) {
+    inconsistentRegions = hbckChecker.getInconsistentRegions();
+    orphanRegionsOnRS = hbckChecker.getOrphanRegionsOnRS();
+    orphanRegionsOnFS = hbckChecker.getOrphanRegionsOnFS();
```

Comment:
Yeah, so this stuff could be an hour old?

Reply:
Yes.

Reply:
Date it? See patch in HBASE-22741. It dates the CJ run Report.

Reply:
There are a date in the latest code.